src/modules/swap/handlers/providers/mew-provider-class.js

```diff
@@ -48,12 +48,21 @@ class MEWPClass {
     return Promise.resolve(isAddress(address));
   }
   getMinMaxAmount({ fromT }) {
+    // If decimals are 0 then min and max will be 1
+    let decimals = fromT.decimals;
+    if (fromT.decimals === 0) {
+      const circulatingSupply = new BigNumber(fromT.market_cap)
+        .dividedToIntegerBy(new BigNumber(fromT.pricef.replace('$', '')))
```

Comment:
can't we use .price here to avoid doing a replace?

Reply:
.price returns a value with only two decimals, DCN .price returns '0.00' since the price is far below a penny.